codecov.yaml

```diff
@@ -0,0 +1,10 @@
+coverage:
+  status: 
+    patch:
+      default:
+        target: 100%
+        threshold: 0.1%
```

Comment:
Are we adding this threshold because the coverage might be irregular? Imo we shouldn't be affected by coverage irregularities and allowing a threshold of 0.1% is not the way to fix it.
We're always uploading our coverage reports, our tests should not be time sensitive etc. If we are having irregularities I'd like to investigate what the root cause of them is instead of allowing for thresholds.